pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/security/StaticFieldMustBeFinal.java

```diff
@@ -0,0 +1,11 @@
+/**
+ * BSD-style license; for more info see http://pmd.sourceforge.net/license.html
+ */
+
+package net.sourceforge.pmd.lang.java.rule.security;
+
+import net.sourceforge.pmd.lang.java.rule.AbstractJavaRule;
+
+public class StaticFieldMustBeFinal extends AbstractJavaRule {
```

Comment:
This class is not used and can be deleted. It's also not a test class so the comment about additional unit tests is misplaced.

OTOH a test class is missing. It should be in the `src/test/java` source tree (not `src/main/java` like here), and extend the base class for rule tests (`PmdRuleTst`).